apps/cli/__init__.py

```diff
@@ -73,6 +73,7 @@ def _lazy_load(self, cmd_name):
             "run-python-step": "etl.run_python_step.main",
             "map-datasets": "apps.utils.map_datasets.cli",
             "scan-chart-diff": "apps.utils.scan_chart_diff.cli",
+            "start-server": "apps.utils.start_server.cli",
```

Comment:
We should think twice about naming it right. I don't have a strong preference and will leave it up to you (`start-server` is good, don't get me wrong!). Here are my 2c:
- I can imagine having other server-specific commands, for instance `refresh-server` or `destroy-server` (these would be just wrappers around `owid-lxc` commands with better instructions). Maybe we could have a group of commands `server start`, `server refresh`, etc.?
- It seems like the important thing here is to create a PR, starting a server is a side effect, so perhaps `create-pr` name would be more intuitive? (this goes against my previous point, feel free to pick one...)

This is a great idea! The only risk is that people will start creating servers like crazy and I'll have to start destroying them sooner than after 7 days (that's gonna change anyway with the chart-diff workflow).

Reply:
Given that, for now, we don't have any other server operations, I've renamed it `etl d draft-pr`. We can merge it for now like this and then discuss it with Lucas on Monday, and possibly rename it.

Reply:
Sounds good!